Anatolie/src/main/java/Month.java

```diff
@@ -0,0 +1,51 @@
+public class Month {
+    public static void printMonthByNumber(int number) {
+        switch (number) {
+            case 1:
+                System.out.println("January");
+                break;
+            case 2:
+                System.out.println("February");
+                break;
+            case 3:
+                System.out.println("March");
+                break;
+            case 4:
+                System.out.println("April");
+                break;
+            case 5:
+                System.out.println("May");
+                break;
+            case 6:
+                System.out.println("June");
+                break;
+            case 7:
+                System.out.println("July");
+                break;
+            case 8:
+                System.out.println("August");
+                break;
+            case 9:
+                System.out.println("September");
+                break;
+            case 10:
+                System.out.println("October");
+                break;
+            case 11:
+                System.out.println("November");
+                break;
+            case 12:
+                System.out.println("December");
+                break;
+            default:
+                System.out.println("Invalid Month");
+        }
+    }
+
+    public static void main(String[] args) {
+        printMonthByNumber(1);
```

Comment:
Good job moving the printMonth logic into its own method.
Keep following this approach in all you further tasks: its always good to have your logic split into separate meaningful methods as it helps to understand your code and makes it reusable.